### ISSUE TITLE
Buggy rules

### ISSUE THREAD

**Issue by bartlomieju** [denoland/deno_lint#154]
False positives found while linting `deno_std` (https://github.com/denoland/deno/pull/6240)

- `no-misued-new`
```
(no-misused-new) Class cannot have method named `new`.
  static create(writer: Writer, size: number = DEFAULT_BUF_SIZE): BufWriter {
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    at ./std/io/bufio.ts:451:2

(no-misused-new) Class cannot have method named `new`.
  static create(
  ~~~~~~~~~~~~~~
    at ./std/io/bufio.ts:540:2
```

- `no-case-declarations`
```
(no-case-declarations) Unexpected declaration in case
      case "string":
      ~~~~~~~~~~~~~~
    at ./std/fmt/printf.ts:631:6

(no-case-declarations) Unexpected declaration in case
        case OpCode.Close:
        ~~~~~~~~~~~~~~~~~~
    at ./std/ws/mod.ts:265:8
```

- `no-empty`
```
(no-empty) Empty block statement
        process.on("uncaughtException", (_err: Error) => {});
                                                         ~~
    at ./std/node/process_test.ts:76:57

(no-empty) Empty block statement
    const testFunction = (): void => {};
                                     ~~
    at ./std/node/events_test.ts:134:37
```

- `no-empty-function`
```
(no-empty-function) Empty functions are not allowed
function callbackify<Arg1T, Arg2T, Arg3T, ResultT>(
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    at ./std/node/_util/_util_callbackify.ts:53:0

(no-empty-function) Empty functions are not allowed
function callbackify<Arg1T, Arg2T, Arg3T, Arg4T, ResultT>(
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    at ./std/node/_util/_util_callbackify.ts:56:0

(no-empty-function) Empty functions are not allowed
function callbackify<Arg1T, Arg2T, Arg3T, Arg4T, Arg5T, ResultT>(
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    at ./std/node/_util/_util_callbackify.ts:70:0
```
